modules/dmrpp_module/data/get_dmrpp.in

```diff
@@ -30,14 +30,15 @@ BES.modules=dap,cmd,h5,dmrpp
 
 # Despite the comment at the top, use the development modules for now. jhrg 5/11/18
 
-# BES.module.dap=@modulesdir@/libdap_module.so
-# BES.module.cmd=@modulesdir@/libdap_xml_module.so
-# BES.module.h5=@modulesdir@/libhdf5_module.so
-
-BES.module.dap=@abs_top_builddir@/dap/.libs/libdap_module.so
-BES.module.cmd=@abs_top_builddir@/xmlcommand/.libs/libdap_xml_module.so
-BES.module.h5=@abs_top_builddir@/modules/hdf5_handler/.libs/libhdf5_module.so
-BES.module.dmrpp=@abs_top_builddir@//modules/dmrpp_module/.libs/libdmrpp_module.so
+BES.module.dap=@bes_modules_dir@/libdap_module.so
+BES.module.cmd=@bes_modules_dir@/libdap_xml_module.so
+BES.module.h5=@bes_modules_dir@/libhdf5_module.so
+BES.module.dmrpp=@bes_modules_dir@/libdmrpp_module.so
+
+#BES.module.dap=@abs_top_builddir@/dap/.libs/libdap_module.so
```

Comment:
I think it would be better to remove these (the commented out lines that reference the modules in the build tree) and the comment on line 31.